src/main/scala/org/constellation/storage/SnapshotService.scala

```diff
@@ -41,6 +41,6 @@ class SnapshotService(implicit dao: DAO) {
       .map(_ => ())
   }
 
-  def lookup: String => IO[Option[Snapshot]] =
-    Lookup.extendedLookup[IO, String, Snapshot](List(midDb, oldDb))
+  def lookup(key: String): IO[Option[Snapshot]] =
+    Lookup.extendedLookup[IO, String, Snapshot](List(midDb, oldDb))(key)
```

Comment:
Could use an intermediate val for readability

Reply:
What do you mean?

Reply:
just to avoid the sequence of brackets `[]()()` one may introduce an intermediate values as in
```scala
def f(key) = { 
    t = g[...](...)
    t(key)
}
```